test/TestUtils/RepoDirectoriesProvider.cs

```diff
@@ -18,6 +18,9 @@ public class RepoDirectoriesProvider
         private string _corehostPackages;
         private string _corehostDummyPackages;
 
+        private string _targetRID;
+
+        public string TargetRID => _targetRID;
```

Comment:
While we do have various environment variables, they always seem a bit fragile to me in favor of having a concrete property to reflect the state.

The directories provider also operates upon a RID specific location and TargetRID corresponds to that (which reflects the intended target of execution).